guides/common/modules/proc_increasing-host-limit-for-pull-based-rex-transport.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Increasing_host_limit_for_pull-based_REX_transport_{context}"]
+= Increasing Host Limit for Pull-Based REX Transport
+
+{Project} has a pull-based transport that has the features of REX but with a deployment model similar to katello-agent. 
```

Comment:
I would not mention "katello-agent" anymore", at least not for Foreman nightly. Maybe you can create a tiny follow-up PR that only gets merged to 3.7 and below?

Reply:
Perhaps this is something that should be handled  as part of https://github.com/theforeman/foreman-documentation/pull/2508?